src/components/TextLink.js

```diff
@@ -32,20 +32,35 @@ const defaultProps = {
 
 const TextLink = (props) => {
     const additionalStyles = _.isArray(props.style) ? props.style : [props.style];
+
+    /**
+     * Checking Function on open TextLink identify whether Clicked or Entered key press
+     *
+     * @param {Event} event
+     * @returns {Function} Returns onPress function.
```

Comment:
```suggestion
     * @param {Event} event
```
The function actually returns `null` or void, so you can skip putting it in the jsdocs

Reply:
agree